analytics.js

```diff
@@ -158,6 +158,28 @@ class Analytics {
     this.readyCallbacks.forEach((callback) => callback());
   }
 
+  /**
+   * A function to validate integration SDK is available in window
+   * and integration constructor is not undefined
+   * @param {string} pluginName
+   * @param {string} modName
+   * @returns boolean
+   */
+  integrationSDKLoaded(pluginName, modName) {
+    try {
+      if (
+        window.hasOwnProperty(pluginName) &&
+        typeof window[pluginName][modName].prototype.constructor !== 'undefined'
+      ) {
+        return true;
+      }
+      return false;
```

Comment:
You can simplify it.
```suggestion
      return (
        window.hasOwnProperty(pluginName) &&
        typeof window[pluginName][modName].prototype.constructor !== 'undefined'
      );
```